### PR TITLE
simplify part settings retrieval for AutoroutePartDisplay

### PR DESCRIPTION
**Pull request by giannik**
#5365

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Autoroute/Drivers/AutoroutePartDisplay.cs [hunk 1]
```diff
@@ -99,37 +89,46 @@ public override async Task<IDisplayResult> UpdateAsync(AutoroutePart model, IUpd
...
+                    S["Your permalink can't be set to the homepage, please use the homepage option instead."]);
             }
 
             if (autoroute.Path?.IndexOfAny(InvalidCharactersForPath) > -1 || autoroute.Path?.IndexOf(' ') > -1)
             {
                 var invalidCharactersForMessage = string.Join(", ", InvalidCharactersForPath.Select(c => $"\"{c}\""));
-                updater.ModelState.AddModelError(Prefix, nameof(autoroute.Path), S["Please do not use any of the following characters in your permalink: {0}. No spaces are allowed (please use dashes or underscores instead).", invalidCharactersForMessage]);
+                updater.ModelState.AddModelError(Prefix, nameof(autoroute.Path),
+                    S[
+                        "Please do not use any of the following characters in your permalink: {0}. No spaces are allowed (please use dashes or underscores instead).",
```
Comment (deanmarcussen) [inline src/OrchardCore.Modules/OrchardCore.Autoroute/Drivers/AutoroutePartDisplay.cs]: why the extra line here?

**Comment by sebastienros** [comment]
Can you revert all the formatting changes?

**Comment by giannik** [comment]
Sorry for that. 
Just saw that you  fixed the formatting.
Will be more careful next time.

**Comment by giannik** [comment]
> Can you revert all the formatting changes?

done, sebastien.

**Pull request merged**